package.json

```diff
@@ -41,7 +41,8 @@
     "test:watch": "jest --watch --runInBand",
     "test:debug": "node --inspect node_modules/.bin/jest --runInBand",
     "type-check": "tsc --pretty --noEmit",
-    "watch": "concurrently --raw --kill-others 'yarn relay --watch' 'yarn compile -w' 'yarn emit-types --watch'"
+    "watch": "concurrently --raw --kill-others 'yarn relay --watch' 'yarn compile -w' 'yarn emit-types --watch' 'node scripts/watch-link.js'",
+    "dev-force": "yarn compile && yarn yalc publish && ./scripts/link-force.sh && yarn watch"
```

Comment:
Why not `link-force`?

Reply:
Just trying to make it representative of what you're trying to do, not how you're trying to do it.

Reply:
How about having `watch` just _always_ publish (or is it really costly?) and then add a package script _to force_ that runs `yalc link @artsy/reaction`? It’s 2 steps instead of 1, but I think it would keep things simpler to understand to the user.

Reply:
Watch does always publish. Ultimately I just want to make this whole process as seamless as possible. Ideally the dev doesn't even have to open force. The only reason I haven't grouped everything into a single command (including starting force) is because the terminal outputs would be mixed and I think that could cause further complications.

Reply:
I hear you, but I’m wondering if having to go into Reaction to link into Force is a helpful level of indirection. The same is going to apply to Volt, Positron, etc.

I think it makes more sense to have the app be responsible for linking and would also allow for the command to perform the linking be the same regardless of what app using Reaction.

If it turns out that people have trouble doing so, we can always add more abstractions later on.